packages.bzl

```diff
@@ -9,6 +9,7 @@ VERSION_PLACEHOLDER_REPLACEMENTS = {
     "0.0.0-MDC": MDC_PACKAGE_VERSION,
     "0.0.0-NG": ANGULAR_PACKAGE_VERSION,
     "0.0.0-TSLIB": TSLIB_PACKAGE_VERSION,
+    "0.0.0-PLACEHOLDER": "{BUILD_SCM_VERSION}",
```

Comment:
Could we add a comment for what this placeholder is used for? The others are already pretty self-explanatory